src/chain/CacheManager.sol

```diff
@@ -107,17 +107,18 @@ contract CacheManager {
     }
 
     /// Places a bid, reverting if payment is insufficient.
-    function placeBid(bytes32 codehash) external payable {
+    function placeBid(address program) external payable {
         if (isPaused) {
             revert BidsArePaused();
         }
+        bytes32 codehash = program.codehash;
```

Comment:
can we either also make codehash a parameter and compare it with `program.codehash` or at least check ` program.codehash != keccak256("")` to make this safer?

Reply:
The latter check is implicit because activation will fail if it's not a Stylus program, but I'd be happy to add a codehash parameter if you think we should have one. That sounds reasonable to me.

Reply:
> latter check is implicit
sgtm then!